### ISSUE TITLE
missing VERSION file causes install to fail

### ISSUE THREAD

**Issue by rleonardibm** [cloudant/python-cloudant#360]
Please include the following information in your ticket.

- Cloudant (python-cloudant) version(s) that are affected by this issue. - 2.8
- Python version - 2.7.10
- A small code sample that demonstrates the issue.

Looks like setup.py is looking for a VERSION file in the install dir and not finding it.

Traceback (most recent call last):
  File "setup.py", line 27, in <module>
    version_file = open('VERSION')
IOError: [Errno 2] No such file or directory: 'VERSION'

**Comment by ricellis** [comment]
Did you install using `pip`?

**Comment by Fokko** [comment]
We experience the same issue with our Airflow environment: https://travis-ci.org/apache/incubator-airflow/jobs/342036090

**Comment by Fokko** [comment]
Very easy to reproduce:
```
Fokkos-MacBook-Pro:incubator-airflow fokkodriesprong$ docker run -t -i python:3 bash
Unable to find image 'python:3' locally
3: Pulling from library/python
Digest: sha256:b93122e0981ce0be50bccfb82a3475bba2b5dcb44dbd4d861c7f6242386a85a9
Status: Downloaded newer image for python:3
root@f1ce60ff7cd8:/# pip install cloudant
Collecting cloudant
  Downloading cloudant-2.8.0.tar.gz (54kB)
    100% |████████████████████████████████| 61kB 2.0MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-jqbv_3rw/cloudant/setup.py", line 27, in <module>
        version_file = open('VERSION')
    FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-jqbv_3rw/cloudant/
```

**Comment by ricellis** [comment]
Hopefully the version file is now included in 2.8.1.

**Comment by rmgpinto** [comment]
For me everything works now. Thanks for the quick update.
